### PR TITLE
Add support for some llvm FP intrinsic in llvm-spirv

### PR DESCRIPTION
**Pull request by jinge90**
Signed-off-by: gejin <ge.jin@intel.com>

### CODE REVIEW

**Comment by jinge90** [comment]
Hi, @bader and @AlexeySotkin 
Who should I turn to for a code review in SPIRV-LLVM-Translator repo?
Thanks very much.

File: test/fp-intrinsics.ll [hunk 1]
```diff
@@ -0,0 +1,346 @@
+; RUN: llvm-as %s -o %t.bc
```
Comment (AlexeySachkov) [inline test/fp-intrinsics.ll:1]: Could you please move this file into `test/llvm-intrinsics` directory?

**Review by AlexeySachkov — APPROVED** [review]
Other than this LGTM

File: test/fp-intrinsics.ll (hunk 1 above)
Comment (jinge90) [reply test/fp-intrinsics.ll]: Done.

**Comment by jinge90** [comment]
It seems that latest llvm-spirv build failed....

**Review by AlexeySotkin — APPROVED** [review]

**Comment by Fznamznon** [comment]
> It seems that latest llvm-spirv build failed....

Build should be fixed by https://github.com/KhronosGroup/SPIRV-LLVM-Translator/pull/856 . Please try to do merge with master.

**Pull request merged**